dev-environment.yml

```
@@ -0,0 +1,9 @@
+name: hist
+channels:
+  - conda-forge
+dependencies:
+  - pip >=18
+  - root
```

```suggestion
  - root >=6.20.0
```

Might speed up the solver a bit.